ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -167,7 +167,7 @@ public func toRACSignal<Value: AnyObject, Error: NSError>(signal: Signal<Value?,
                 subscriber.sendError(error)
             case .Completed:
                 subscriber.sendCompleted()
-            default:
+			case .Interrupted:
```

Comment:
This line displays perfectly well on Xcode, I don't know why it got shifted on GitHub...